backend/src/main/java/com/woowacourse/gongseek/comment/presentation/CommentController.java

```diff
@@ -0,0 +1,62 @@
+package com.woowacourse.gongseek.comment.presentation;
+
+import com.woowacourse.gongseek.auth.presentation.AuthenticationPrinciple;
+import com.woowacourse.gongseek.auth.presentation.dto.LoginMember;
+import com.woowacourse.gongseek.comment.application.CommentService;
+import com.woowacourse.gongseek.comment.presentation.dto.CommentRequest;
+import com.woowacourse.gongseek.comment.presentation.dto.CommentResponse;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping("/api/articles")
+@RequiredArgsConstructor
```

Comment:
```suggestion
@RequiredArgsConstructor
@RequestMapping("/api/articles")
```
모든 생성자 생성 어노테이션을 맨 상단에 두는건 어떤가요?😃